### DISCUSSION TITLE
VSCode Source Control Repositories and Commit not working

### DISCUSSION THREAD

**Discussion by jnewco11** [giswqs/geog-312#35] [Q&A] [answered]
![Screenshot 2024-08-30 074734](https://github.com/user-attachments/assets/f232d9ad-4b02-4fe7-939d-16caee59edcc)
![Screenshot 2024-08-30 074803](https://github.com/user-attachments/assets/8062226d-38d2-4039-bc6d-3b7ff07d29ae)
In Visual Studio Code, I tried to use the Source Control Repositories as shown in the newest video. However, I ran into an issue. As shown in the screenshots, it does not show anything but the book folder --> 02 variables. I'm not sure how to use Commit either, it does not work on my VSCode as shown in the video. The main issue is that VSCode will not update to new changes or added content. Is there anyway to fix this?

**Comment by giswqs** [comment, marked as answer]
Step 1: Discard all local changes
![image](https://github.com/user-attachments/assets/398161ff-9571-45fd-b90d-551bd063351c)

Step 2: Git Pull
![image](https://github.com/user-attachments/assets/d7b20ab1-de6a-4dd0-a12b-41848234d100)

If it does not work, delete your local repository, and clone again.

**Reply by jnewco11** [reply]
I think it worked. Thank you.
